Makefile

```diff
@@ -35,10 +35,10 @@ endef
 export SDKUsageJavascript
 
 test-for-release:
-	npm install && npm pack
-	npx react-native init ${RELEASE_TEST_APP_NAME}
-	# dd-sdk-reactnative-0.1.0.tgz is hardcoded for now, ideally we should read the version from package.json
-	cd ${RELEASE_TEST_APP_NAME} && npm install --save ../dd-sdk-reactnative-0.1.0.tgz
+	yarn install && yarn workspace @datadog/react-native-sdk pack
+	npx react-native init ${RELEASE_TEST_APP_NAME} --version 0.63.4
+	# datadog-react-native-sdk-v1.0.0-beta4.tgz is hardcoded for now, ideally we should read the version from package.json
+	cd ${RELEASE_TEST_APP_NAME} && npm install --save ../packages/core/datadog-react-native-sdk-v1.0.0-beta4.tgz
```

Comment:
i just saw [code to read version from `package.json` here](https://github.com/DataDog/datadog-ci/blob/master/src/commands/sourcemaps/upload.ts#L112), maybe we can use that. 
```
const cliVersion = require('../../../package.json').version
```
i'm leaving this as an 💡

Reply:
yes, but we need to get in the shell script. maybe there is some JS REPL though, like from `node`, which we can use in `.sh`. I will check it out.

Reply:
Found a way to do it https://github.com/DataDog/dd-sdk-reactnative/pull/79/commits/148465cee870b234ae14db71189b842088447db8

Reply:
super 💯